tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -499,57 +500,77 @@ Section WebView2
+        DetailPrint "$(installingWebview2)"
+        StrCpy $6 "$TEMP\MicrosoftEdgeWebview2Setup.exe"
+        Goto install_webview2
+      !endif
+
+      !if "${INSTALLWEBVIEW2MODE}" == "offlineInstaller"
+        Delete "$TEMP\MicrosoftEdgeWebView2RuntimeInstaller.exe"
+        File "/oname=$TEMP\MicrosoftEdgeWebView2RuntimeInstaller.exe" "${WEBVIEW2INSTALLERPATH}"
+        DetailPrint "$(installingWebview2)"
+        StrCpy $6 "$TEMP\MicrosoftEdgeWebView2RuntimeInstaller.exe"
+        Goto install_webview2
+      !endif
+
+      Goto webview2_done
+
+      install_webview2:
+        DetailPrint "$(installingWebview2)"
+        ; $6 holds the path to the webview2 installer
+        ExecWait "$6 ${WEBVIEW2INSTALLERARGS} /install" $1
+        ${If} $1 = 0
+          DetailPrint "$(webview2InstallSuccess)"
+        ${Else}
+          DetailPrint "$(webview2InstallError)"
+          Abort "$(webview2AbortError)"
+        ${EndIf}
+      webview2_done:
+    ${EndIf}
+  ${Else}
+    !if "${MINIMUMWEBVIEW2VERSION}" != ""
+      ${If} $4 != ""
+        ${VersionCompare} "${MINIMUMWEBVIEW2VERSION}" "$4" $R0
       ${Else}
-        DetailPrint "$(webview2DownloadError)"
-        Abort "$(webview2AbortError)"
+        ${VersionCompare} "${MINIMUMWEBVIEW2VERSION}" "$5" $R0
       ${EndIf}
-      StrCpy $6 "$TEMP\MicrosoftEdgeWebview2Setup.exe"
-      Goto install_webview2
-    !endif
-
-    !if "${INSTALLWEBVIEW2MODE}" == "embedBootstrapper"
-      Delete "$TEMP\MicrosoftEdgeWebview2Setup.exe"
-      File "/oname=$TEMP\MicrosoftEdgeWebview2Setup.exe" "${WEBVIEW2BOOTSTRAPPERPATH}"
-      DetailPrint "$(installingWebview2)"
-      StrCpy $6 "$TEMP\MicrosoftEdgeWebview2Setup.exe"
-      Goto install_webview2
-    !endif
-
-    !if "${INSTALLWEBVIEW2MODE}" == "offlineInstaller"
-      Delete "$TEMP\MicrosoftEdgeWebView2RuntimeInstaller.exe"
-      File "/oname=$TEMP\MicrosoftEdgeWebView2RuntimeInstaller.exe" "${WEBVIEW2INSTALLERPATH}"
-      DetailPrint "$(installingWebview2)"
-      StrCpy $6 "$TEMP\MicrosoftEdgeWebView2RuntimeInstaller.exe"
-      Goto install_webview2
-    !endif
-
-    Goto webview2_done
-
-    install_webview2:
-      DetailPrint "$(installingWebview2)"
-      ; $6 holds the path to the webview2 installer
-      ExecWait "$6 ${WEBVIEW2INSTALLERARGS} /install" $1
-      ${If} $1 = 0
-        DetailPrint "$(webview2InstallSuccess)"
-      ${Else}
-        DetailPrint "$(webview2InstallError)"
-        Abort "$(webview2AbortError)"
+      ${If} $R0 = 1
+        DetailPrint "$(installingWebview2)"
+        ; Chromium updater docs: https://source.chromium.org/chromium/chromium/src/+/main:docs/updater/user_manual.md
+        ; Modified from "HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft EdgeWebView\ModifyPath"
+        ExecWait `"C:\Program Files (x86)\Microsoft\EdgeUpdate\MicrosoftEdgeUpdate.exe" /install appguid={F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}&needsadmin=true` $1
```

Comment:
I don't like this hardcoded value, especially since `C:/` is not guaranteed to be the OS drive. We should check `path` key inside `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate`

Reply:
Right, I can take a look at it this Friday probably, kinda busy working on a game mod right now, releasing in a few days

Reply:
no worries, whenever you have time